jenkins_job_wrecker/modules/properties.py

```diff
@@ -116,18 +117,28 @@ def parameters(top, parent):
             for setting in param:
                 key = {'defaultValue': 'default'}.get(setting.tag, setting.tag)
                 if setting.text is None:
-                    parameter[key] = ''
+                    if setting.tag != "description" or param_type != "parameter-separator":
```

Comment:
I don't get this condition. I've commented it out and the tests still pass. Is it still needed?

Reply:
It must have forgotten to remove it during debugging, it does not make any sense for me as well.